### DISCUSSION TITLE
hook-importlib_metadata.py is missing

### DISCUSSION THREAD

**Discussion by karthikbalasubramani** [pyinstaller/pyinstaller#8803] [Help]
After installing pyInstaller i hope this folder will be created inside site-packagaes **site-packages\_pyinstaller_hooks_contrib\stdhooks** but unfortunately that folder doesn't have the hook-importlib_metadata.py file but it is present in **site-packages\PyInstaller\hooks**

I am getting issue in Jenkins:

**pkg_resources.DistributionNotFound: The 'importlib_metadata' distribution was not found and is required by the application
Info - version updated to: 1.0.33812.0**

Note: Environment Windows

**Comment by bwoodsend** [comment]
> After installing pyInstaller i hope this folder will be created inside site-packagaes site-packages_pyinstaller_hooks_contrib\stdhooks but unfortunately that folder doesn't have the hook-importlib_metadata.py file but it is present in site-packages\PyInstaller\hooks

It only needs to go in one or the other.

Can we have some proper diagnostics? Full error message, minimal reproducer, versions of PyInstaller and importlib_metadata?
